drivers/sensor/iis2iclx/iis2iclx_trigger.c

```diff
@@ -172,7 +166,7 @@ static void iis2iclx_gpio_callback(const struct device *dev,
 
 	ARG_UNUSED(pins);
 
-	gpio_pin_interrupt_configure(iis2iclx->gpio, cfg->irq_pin,
+	gpio_pin_interrupt_configure(cfg->gpio_drdy.port, cfg->gpio_drdy.pin,
```

Comment:
We could use `gpio_pin_interrupt_configure_dt` here and a few other places in this PR.

Reply:
Ack